pkg/restapi/vc/operation/operations_test.go

```diff
@@ -20,11 +20,16 @@ import (
 	"testing"
 	"time"
 
+	"github.com/btcsuite/btcutil/base58"
+
 	"github.com/gorilla/mux"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
+	"github.com/hyperledger/aries-framework-go/pkg/kms/legacykms"
 	kmsmock "github.com/hyperledger/aries-framework-go/pkg/mock/kms/legacykms"
 	vdrimock "github.com/hyperledger/aries-framework-go/pkg/mock/vdri"
+	ariesstorage "github.com/hyperledger/aries-framework-go/pkg/storage"
```

Comment:
Aries storage is used here for manipulating KMS objects which exist in some tests. I believe the edge-core storage technically has the same signature, so they might appear to be compatible interfaces but the semantics are different due to the separate of Open/Create store in the edge-core version.